Newer versions of abseil and re2 require C++17, so we need to patch effcee to not hardcode C++11
--- spirv-tools/external/effcee/cmake/utils.cmake.orig	2025-06-20 17:40:20.428533870 +0200
+++ spirv-tools/external/effcee/cmake/utils.cmake	2025-06-20 17:40:32.853696425 +0200
@@ -51,7 +51,7 @@
   effcee_default_c_compile_options(${TARGET})
   if (NOT "${MSVC}")
     # RE2's public header requires C++11.  So publicly required C++11
-    target_compile_options(${TARGET} PUBLIC -std=c++11)
+    target_compile_options(${TARGET} PUBLIC -std=c++17)
     if (NOT EFFCEE_ENABLE_SHARED_CRT)
       if (WIN32)
         # For MinGW cross compile, statically link to the C++ runtime.
